### PR TITLE
release-0.0.5

### PR DESCRIPTION
**Pull request by bhansaliraunak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
-# Environmental-Intelligence
-IBM Environmental Intelligence documentation
+# IBM Environmental Intelligence
+
+This repository contains sample applications of IBM Environmental Intelligence. 
+
+### General Notes
+
+If you like to contribute, please read [CONTRIBUTING.md](CONTRIBUTING.md)
```
Comment (IvanLew) [inline README.md:7]: Change "If you like" to "If you would like"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+[Water and Moisture Detection](geospatial_analytics/v3_apis/samples/industry_use_cases/agriculture_water_and_moisture_detection/water_and_moisture_detection.ipynb) | Normalized Difference Moisture Index (NDMI) and Modified Normalized Difference Water Index (mNDWI) provides a relative indication of the wetness or dryness of an area, helping to identify potential water stress or drought conditions or combustibility levels in fire-prone areas. | Agriculture, Disaster Events
+[Sea Rise Risk Prediction](geospatial_analytics/v3_apis/samples/industry_use_cases/climate_change_tidal_surge/sea_rise_risk_prediction.ipynb) | Geospatial Analytics can help determine the possible regions that come under the threat of rise in sea levels                                                                                                                                                                          | Climate Change, Disaster Events
+
+## Geospatial Analytics Above Ground Biomass (AGB)
+
+To know the basics of Geospatial Analytics Above Ground Biomass (AGB) Dataset, start with the [AGB Catalogue](geospatial_analytics/v4_apis/samples/quickstart/agb_catalogue/agb_catalogue.ipynb) sample.
```
Comment (IvanLew) [inline README.md:23]: Change "To know the basics" to "To understand the basics"

**Review by IvanLew — COMMENTED** [review]
Added a few comments

**Review by rphadnis — APPROVED** [review]

**Review by IvanLew — APPROVED** [review]
Approve to get sample out. Minor comments will be updated in subsequent PRs

**Pull request merged**